README.md

```diff
@@ -79,21 +106,26 @@ export QPC_DBMS=SQLite
 ```
 
 ## Initializing the Server
-To initialize the server with Postgres, run the following command:
+To initialize the server with SQlite, run the following command:
 ```
-make server-init
+make server-init -e QPC_DBMS=sqlite
 ```
 
 Both of the above commands create a superuser with name `admin` and password of `qpcpassw0rd`.
 
 ## Running the Server
-To run the development server using Postgres, run the following command:
+Currently, quipucords needs quipucords-ui to run, so assuming you are on the root directory of the repository, /quipucords-ui must be relative to /quipucords.
```

Comment:
This one got me, initially. It may be more helpful to show the directory structure to avoid confusion
```
/quipucords
    |--/quipucords
    |--/quipucords-ui
```